main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.start
```

Comment:
- [ ] After fixing that import issue your app does not start when I run `ruby main.rb`. If you inspect your `app.rb` file you will find that there is no method called `start` there. Kindly run your app ensure there no errors.

  ![image](https://github.com/mohamedgamaleldin999999/School-library/assets/36542986/4f992a58-dbe1-4ff2-b4b8-2c63b4cebde0)